pywt/data/_readers.py

```diff
@@ -141,9 +141,12 @@ def ecg():
 
 def nino():
     """
-    sea surface temperature NINO3 1871-1996 (in C; from the UKMO GISST2.3):
-    Seasonally-averaged, minus the annual cycle
-
+    This data contains the averaged monthly sea surface temperature in degrees 
+    Celcius of the Pacific Ocean, between 0-10 degrees South and 90-80 degrees West, from 1950 to 2016. 
+    This dataset was obtained from NOAA.
```

Comment:
LGTM.  only suggestion would be to specifically mention "public domain" here
e.g.
"This dataset is in the public domain and was obtained from NOAA."